queries.sql

```diff
@@ -0,0 +1,8 @@
+SELECT * FROM animals WHERE animal_name LIKE '%mon%';
+SELECT * FROM animals WHERE date_of_bird >= '01-01-2016'::date AND date_of_bird <= '12-31-2019'::date;
+SELECT * FROM animals WHERE neutered IN (true) AND escape_attempts < 3;
+SELECT date_of_bird FROM animals WHERE animal_name IN ('Agumon', 'Pikachu');
```

Comment:
Please correct the typo here as it is `date_of_birth` not `date_of_bird`.

Reply:
Ops 😖 , my bad, thanks for your review!!